examples/wave/wave-op-mpi.py

```diff
@@ -57,7 +57,9 @@
 
 # {{{ wave equation bits
 
-@with_container_arithmetic(bcast_obj_array=True, rel_comparison=True)
+@with_container_arithmetic(bcast_obj_array=True, rel_comparison=True,
+        # FIXME: I'll put together an actx patch to make this unnecessary
+        _cls_has_array_context_attr=True)
```

Comment:
We can delete this once https://github.com/inducer/arraycontext/pull/155 is in.

Reply:
Alas, it is not to be. See the discussion in https://github.com/inducer/arraycontext/pull/155.